calculators/referendum_supporters_calculator.rb

```diff
@@ -29,6 +39,37 @@ def fetch
       end
     end
 
+    late_expenditures.each do |row|
+      sup_opp_cd = guess_whether_committee_supports_measure(row['Filer_ID'], row['Bal_Name'])
```

Comment:
Yep, we drop it. I created #35 as a issue to track our tracking of how many records / how much money we skip because we can't find a match. We can keep adding more to that issue as we introduce more fuzzy matching into our processing.